widget-src/components/log/User.tsx

```diff
@@ -6,59 +6,60 @@ const { AutoLayout, Image, Rectangle, Text } = widget;
 interface UserProps {
   userName: string | undefined;
   userPhotoUrl: string | null | undefined;
+  showAvatars: boolean;
 }
 
-export const User = ({ userName, userPhotoUrl }: UserProps) => {
+export const User = ({ userName, userPhotoUrl, showAvatars }: UserProps) => {
   return (
     <AutoLayout
       name="User"
       overflow="visible"
       direction="vertical"
       spacing={GAP.none}
-      padding={{
-        vertical: PADDING.xl,
-        horizontal: PADDING.none,
-      }}
+      padding={
+        showAvatars
+          ? {
+              vertical: PADDING.xl,
+              horizontal: PADDING.none,
+            }
+          : {
+              vertical: PADDING.xl,
+              right: PADDING.sm,
+              left: PADDING.none,
+            }
+      }
```

Comment:
```suggestion
      padding={{
        vertical: PADDING.xl,
        horizontal: PADDING.none,
        right: showAvatars ? PADDING.none : PADDING.sm,
      }}
```
Bit of a nit, but this is less repetitive and seems to work just fine